server/lib/models/account.go

```diff
@@ -77,6 +71,19 @@ func (a *Account) StringId() string {
 	return strconv.FormatUint(uint64(a.Id), ACCOUNT_ID_BASE)
 }
 
+func (a *Account) Persona(dbif Dbif, channel proto.UUID) (*Persona, error) {
```

Comment:
__[gometalinter]__
* [golint] exported method Account.Persona should have comment or be unexported
* [gotype] undeclared name: proto